packages/tools/src/utilities/planar/filterAnnotationsForDisplay.ts

```diff
@@ -7,8 +7,8 @@ import {
 } from '@cornerstonejs/core';
 
 import filterAnnotationsWithinSlice from './filterAnnotationsWithinSlice';
-import { Annotations } from '../../types';
-import { annotationFrameRange } from '..';
+import type { Annotations } from '../../types';
+import annotationFrameRange from '../annotationFrameRange';
```

Comment:
This line caused the bug - we depended on that class, which depended on the import, which depended on this class - throwing an exception.  Why they couldn't detect that I don't know.